components/NavBar/NavBar.jsx

```diff
@@ -1,52 +1,81 @@
 import React, { useState } from "react";
 import styles from "./NavBar.module.scss";
 import Link from "next/link";
-
+import Alert from "@material-ui/lab/Alert";
 import DrawerToggleButton from "components/SideDrawer/DrawerToggleButton";
+import InfoIcon from "@material-ui/icons/Info";
+import { createMuiTheme, ThemeProvider } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles({
```

Comment:
This pattern of styling components is widely adopted but since our project uses Sass, let's just stick to that. All the styling should be done in NavBar.module.scss